contracts/test/integration/utils/PCVGuardVerification.sol

```diff
@@ -53,8 +53,8 @@ contract PCVGuardVerification is DSTest {
     IERC20[] private allMainnetTokens = [
         IERC20(MainnetAddresses.USDC),
         IERC20(MainnetAddresses.FEI),
-        IERC20(MainnetAddresses.DAI),
-        IERC20(MainnetAddresses.VOLT)
+        IERC20(MainnetAddresses.DAI)
+        // IERC20(MainnetAddresses.VOLT)
```

Comment:
is this failing due to a burn?

Reply:
if it isn't failing, would prefer to keep it in

Reply:
yes, this is failing due to the burn

Reply:
Ok, once the Volt V2 token is deployed, let's add it here, and that should cause the framework to stop yelling.